Frameworks/UIKit/StarboardXaml/ApplicationMain.mm

```diff
@@ -106,12 +108,44 @@ int ApplicationMainStart(const char* principalName,
 
     [displayMode _updateDisplaySettings];
 
-    UIApplicationMainInit(principalClassName, delegateClassName, defaultOrientation, (int)activationType, activationArgument);
+    UIApplicationMainInit(principalClassName, delegateClassName, defaultOrientation, (int)activationType, activationArg);
     ScheduleMainRunLoop();
 
     return 0;
 }
 
+int ApplicationMainStartHSTRING(const char* principalName,
+                                const char* delegateName,
+                                float windowWidth,
+                                float windowHeight,
+                                ActivationType activationType,
+                                HSTRING activationArg) {
+    return _ApplicationMainStart(principalName,
```

Comment:
super nit: would love a newline before the return to just visually separate the function parameters from the  called function args.